mappings/net/minecraft/client/render/entity/LivingEntityRenderer.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS net/minecraft/class_922 net/minecraft/client/render/entity/LivingEntityRenderer
+	FIELD field_21011 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_4737 model Lnet/minecraft/class_583;
 	FIELD field_4738 features Ljava/util/List;
+	METHOD method_18656 getHorizontalAngle (Lnet/minecraft/class_2350;)F
```

Comment:
What's the "horizontal angle"? Isn't it just pitch?

Reply:
It returns an Angle from a Direction. `North` produces 270 - is that equivalent to pitch?

Reply:
It's yaw then. Horizontal angle is confusing, it can mean either pitch (angle between entity and horizontal plane) or yaw (angle on the horizontal plane)

Reply:
ok 😃